src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -953,6 +953,10 @@ sap.ui.define([
 		}
 		if (!vServerValue && oRequestInfo.method === HTTPMethod.PUT) {
```

Comment:
We could create a new private function `_isServerDateUpToDate` where we check if an update is needed

``
FHIRModel.prototype._isServerDateUpToDate = function(vServerValue, oResource, sHTTPMethod){
  if (!vServerValue && sHTTPMethod=== HTTPMethod.PUT) {
    return false;
  } else if (vServerValue &&  sHTTPMethod === HTTPMethod.PUT && deepEqual(vServerValue, oResource)) {
    return false;
  }
  return true;
}
``

and in the `_handleClientChanges ` method we have then:
``
if(!this._isServerDateUpToDate(vServerValue, oResource, oRequestInfo.method)){
  this._setProperty(this.oDataServerState, aResPath, FHIRUtils.deepClone(oResource), true);
}
``

Reply:
done!